Foundation/NSGeometry.swift

```diff
@@ -33,7 +33,11 @@ public struct CGFloat {
     public var native: NativeType
     
     private var hash: Int {
+#if arch(i386) || arch(arm)
```

Comment:
I wish we had a #if 32bit or some sort of equiv; ala the discussion on swift-evolution.